src/features/configPlugins/schema.ts

```diff
@@ -43,7 +43,12 @@ export function getSchemas(): Record<string, (Joi: JoiRoot) => any> {
       }).optional(),
     extraRemarkPlugins: getUnifiedPluginSchema,
     extraRehypePlugins: getUnifiedPluginSchema,
-    themeConfig: (Joi) => Joi.object().optional(),
+    themeConfig: (Joi) =>
+      Joi.object({
+        logo: Joi.alternatives(Joi.string(), Joi.equal(false)).optional(),
+      })
+        .unknown()
+        .optional(),
```

Comment:
抱歉是我搞混了，以为 logo 在配置项顶层；schema 保持现状即可，我直接更新掉了：

```suggestion
    themeConfig: (Joi) => Joi.object().optional(),
```